js_modules/dagster-ui/packages/ui-core/src/app/QueryRefresh.tsx

```diff
@@ -122,9 +126,13 @@ export function useQueryRefreshAtInterval(
       nextFireMs,
       nextFireDelay,
       networkStatus: queryResult.networkStatus,
-      refetch: queryResult.refetch,
+      refetch: (...props) => {
+        return customRefetchRef.current
+          ? (customRefetchRef.current() as any)
```

Comment:
I don't suppose we can do better than `any` here?

Reply:
hmm, I can probably thread through the refresh state of the current batched fetch but that might lead to subtle bugs if anyone relies on it so I'd rather there be a hard error for that case if anyone tried to rely on it. This is probably just temporary though until I come up with something more robust for the asset overview page.